src/js/components/search/filters/timePeriod/TimePeriod.jsx

```diff
@@ -34,15 +22,20 @@ export default class TimePeriod extends React.Component {
         this.state = {
```

Comment:
What about fiscal years? The top filter component's state doesn't seem to store that information/it's not bubbled up from the child components.